_includes/regions/banner.html

```diff
@@ -0,0 +1,8 @@
+<div class="col-md-12">
+  <div class="clearfix wrapper">
+    <p class="headline">This camp is brought to you by the <a href="https://www.facebook.com/groups/drupalcebu/">Drupal Cebu Users Group</a>.</p>
+  </div>
+</div>
+<div class="camp-logo">
+  <img src="{{ "/images/drupalcebu_puso.svg" | relative_url }}" alt="Drupalcamp Cebu 2017 logo" width="125px" height="182px" />
```

Comment:
Are the width and height proportional since they're static?

Reply:
Can you check? I haven't changed that line I think. Just moved files.

Reply:
Ah! Wasn't able to see that. Based on previous developments, I didn't see some distortions in the image's proportion so I think this will be alright as is. Thanks!